floris/convert_floris_input_v3_to_v4.py

```diff
@@ -64,10 +70,20 @@
     # Add enable_active_wake_mixing field
     v4_floris_input_dict["wake"]["enable_active_wake_mixing"] = False
 
-    yaml.dump(
-            v4_floris_input_dict,
-            open(output_path, "w"),
-            sort_keys=False
-        )
+    yaml.dump(v4_floris_input_dict, open(output_path, "w"), sort_keys=False)
+
+    # Open the output file and loop through line by line
+    # if a line contains the substring !include, then strip all
+    # occurrences of ' from the line
```

Comment:
This comment could perhaps state that the yaml load / dump sequence adds the string marks ' ' to the !include statements, which then need to be removed from the final output file. I didn't follow that at first. Otherwise, this looks great!